packages/admin/cms-admin/src/redirects/createRedirectsPage.tsx

```diff
@@ -41,7 +42,8 @@ function createRedirectsPage({ customTargets, scopeParts = [] }: CreateRedirects
             <Stack topLevelTitle={intl.formatMessage({ id: "comet.pages.redirects", defaultMessage: "Redirects" })}>
                 <StackSwitch initialPage="redirectsTable">
                     <StackPage name="redirectsTable">
```

Comment:
```suggestion
                <StackSwitch initialPage="grid">
                    <StackPage name="grid">
```

This should be grid as well.